src/lib/components/copyable.svelte

```diff
@@ -33,17 +35,37 @@
   };
 </script>
 
-<div class="flex gap-2 items-center group {$$props['container-class']}">
-  <slot>
-    <span class={$$props.class} class:select-all={!$$slots.default}
-      >{content}</span
-    >
-  </slot>
-  <button on:click|preventDefault|stopPropagation={copy}>
+{#if clickAllToCopy}
```

Comment:
I don't love the giant conditional, maybe two components is better? But, this is fine for now.

Reply:
Yeah I don't either, but the code is so similar felt like it should stay in same file. If only you could export multiple components in one file...